service/src/main/java/bio/terra/workspace/db/ApplicationDao.java

```diff
@@ -192,19 +192,6 @@ public WsmWorkspaceApplication enableWorkspaceApplication(
     return enableWorkspaceApplicationWorker(workspaceUuid, applicationId);
   }
 
-  /**
-   * Enable a list of applications. Used as part of create workspace
-   *
-   * @param workspaceUuid workspace to enable applications in
-   * @param applicationIds list of application ids
-   */
-  @WriteTransaction
-  public void enableWorkspaceApplications(UUID workspaceUuid, List<String> applicationIds) {
-    for (String applicationId : applicationIds) {
-      enableWorkspaceApplication(workspaceUuid, applicationId);
```

Comment:
this line was the main problem for https://broadworkbench.atlassian.net/browse/WOR-1256. This is called in a function annotated with @WriteTransaction and then it calls a function in another dao that is also @WriteTransaction. The transaction propagation is REQUIRED so the same transaction is used but another retry interceptor is layered in! So we do retries within the same transaction instead of retrying the entire transaction.